sse2neon.h

```diff
@@ -2862,6 +2862,14 @@ FORCE_INLINE __m128i _mm_min_epi16(__m128i a, __m128i b)
         vminq_s16(vreinterpretq_s16_m128i(a), vreinterpretq_s16_m128i(b)));
 }
 
+// Computes the pairwise maxima of the 16 signed 8-bit integers from a and the
+// 16 signed 8-bit integers from b.
+FORCE_INLINE __m128i _mm_max_epi8(__m128i a, __m128i b)
```

Comment:
Add hyperlink to Intel website.

Reply:
Use the description of the hyperlink as well.